src/vaults/cross-chain-vault/router/ERC4626Router.sol

```diff
@@ -41,25 +41,21 @@ contract ERC4626Router is ERC4626RouterBase {
 
     function depositMax(IERC4626 vault, address to, uint256 minSharesOut) public payable returns (uint256 sharesOut) {
         ERC20 asset = ERC20(vault.asset());
-        uint256 assetBalance = asset.balanceOf(_msgSender());
+        uint256 assetBalance = asset.balanceOf(msg.sender);
         uint256 maxDeposit = vault.maxDeposit(to);
         uint256 amount = maxDeposit < assetBalance ? maxDeposit : assetBalance;
-        ERC20(vault.asset()).safeTransferFrom(_msgSender(), address(this), amount);
+        ERC20(vault.asset()).safeTransferFrom(msg.sender, address(this), amount);
         return deposit(vault, to, amount, minSharesOut);
     }
 
     function redeemMax(IERC4626 vault, address to, uint256 minAmountOut) public payable returns (uint256 amountOut) {
-        uint256 shareBalance = ERC20(address(vault)).balanceOf(_msgSender());
-        uint256 maxRedeem = vault.maxRedeem(_msgSender());
+        uint256 shareBalance = ERC20(address(vault)).balanceOf(msg.sender);
+        uint256 maxRedeem = vault.maxRedeem(msg.sender);
         uint256 amountShares = maxRedeem < shareBalance ? maxRedeem : shareBalance;
         return redeem(vault, to, amountShares, minAmountOut);
     }
 
     function approve(ERC20 token, address to, uint256 amount) public payable {
         token.safeApprove(to, amount);
     }
-
-    function versionRecipient() external view virtual override returns (string memory) {
```

Comment:
is this no longer used?

Reply:
its from relayer. dropping relayer from router too.